middleware.ts

```
@@ -0,0 +1,38 @@
+import { NextResponse } from "next/server";
+import type { NextRequest } from "next/server";
+
+export function middleware(request: NextRequest) {
+  // Handle preflight requests
+  if (request.method === "OPTIONS") {
+    const preflightHeaders = {
+      "Access-Control-Allow-Credentials": "true",
+      "Access-Control-Allow-Origin": process.env.NEXT_PUBLIC_API_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)